CONTRIBUTING.md

```diff
@@ -128,12 +127,13 @@ sudo apt update
 sudo apt install build-essential curl openssl libssl-dev pkg-config protobuf-compiler cmake
 
 # Install Go
-export GO_VERSION="1.23.4"
+export GO_VERSION="1.24.0"
+export ARCH="amd64" # E.g. arm64. See https://golang.org/dl/ for available architectures
```

Comment:
Would be nice if we could auto-detect this somehow.

Reply:
Done. The script detects only Amd64 and Arm64 for now.